nxt_editor/node_graphics_item.py

```diff
@@ -452,7 +452,8 @@ def draw_title(self, painter, lod=1.):
 
         # draw collapse state arrow
         for arrow in self.collapse_arrows:
-            self.scene().removeItem(arrow)
+            if arrow.scene():
+                self.scene().removeItem(arrow)
```

Comment:
I assume we can be sure the arrow's scene is _always_ the graphic item's scene?

Reply:
Yes to both, `self.collapse_arrows` will only ever have arrows from the same scene as this node.